example/index.html

```diff
@@ -316,7 +312,10 @@
         if (clickedTrackID === STATE.currentTrackID) {
           togglePlayerState();
         } else {
-          playTrack(TRACKS[clickedTrackID]);
+          player.playTrack(TRACKS[clickedTrackID]);
```

Comment:
`playTrack` is actually not deprecated in this API. I decided to keep a version of it exposed so that it was possible to use the library to play single audio files and/or build a custom queueing/track management system on top of it.

Reply:
That said, I do have a separate example that uses the new API that is fully updated and working. It is the gapless demo page I made, though, and I want to make sure it's okay with Jason that we expose his mix on a public Github demo page.

Reply:
I talked to Jason and it is fine. The example can be found under `example/playlist.html`.

Reply:
Perfect! Thanks for clarifying :)